### PR TITLE
Add supported Python versions to the build matrix

### LINKED ISSUE #1: Test against all supported Python versions on Travis CI
- [ ] Update `.travis.yml`
- [ ] Update `pyproject.toml` for supported versions
- [ ] Update `README.md` for supported versions

### PR DESCRIPTION
**Pull request by jacebrowning**
As long as we plan to support legacy Python, we might as well build against all versions of Python that are still supported by the PSF until we use Python 3.6-only features.

Closes #1

### CODE REVIEW

**Comment by jacebrowning** [comment]
Oh, we're not actually compatible with legacy Python (syntax errors in dependencies). Maybe we should only support Python 3.6+?

**Comment by craiglabenz** [comment]
There shouldn't be any dependencies for `jsonmask` 🤔

**Comment by craiglabenz** [comment]
It looks like a `confest.py` -> `log` error. None of that should be a blocker for the full suite of supported Python versions, IMO.

File: jsonmask/tests/test_mask.py [hunk 1]
```diff
@@ -6,9 +6,6 @@
 
 
 def test_apply_json_mask():
-
-    print('omg')
```
Comment (jacebrowning) [inline jsonmask/tests/test_mask.py:10]: 😬

**Pull request merged**